src/Monolog/Formatter/LogzIoFormatter.php

```diff
@@ -0,0 +1,50 @@
+<?php declare(strict_types=1);
+
+/*
+ * This file is part of the Monolog package.
+ *
+ * (c) Jordi Boggiano <j.boggiano@seld.be>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Monolog\Formatter;
+
+/**
+ * Encodes message information into JSON in a format compatible with Logz.io.
+ *
+ * @author Christian Brückner <chris@chrico.info>
+ */
+class LogzIoFormatter extends JsonFormatter
+{
+
+    /**
+     * yyyy-MM-dd'T'HH:mm:ss.SSSZ
+     */
+    const DATETIME_FORMAT = "c";
```

Comment:
Howdy @martijncalker !

Thanks for your feed. I was just using the format they're suggesting here: https://support.logz.io/hc/en-us/articles/210206885-How-can-I-get-Logz-io-to-read-the-timestamp-within-a-JSON-log- .

Shall i change the pull request to fit into your format? :-)